src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
```

Comment:
Missed requirement: Create `getPostComments(postId)` method in `/src/api/comments.ts`, load comments from `/comments?postId={postId`}.

```suggestion
export async function getPostComments(postId: number) {
  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
```